### PR TITLE
change license to be compliant with bits removed from `ring`

### LINKED ISSUE #1: License text in README.md is wrong
README.md says "This work is dual licensed under the MIT and Apache-2.0 licenses. see LICENSE-MIT and LICENSE-APACHE for details," but at least a significant chunk of this is licensed under the ISC license.

### PR DESCRIPTION
**Pull request by heartsucker**
fixes #1

### CODE REVIEW

**Comment by erickt** [comment]
Hello! What's the status of this? Are you still planning on switching the license?

**Comment by heartsucker** [comment]
Yes, it seems so. I think I'm in violation otherwise. Do you have any input on this?

File: README.md [hunk 1]
```diff
@@ -11,5 +11,4 @@ If you like this lib, thank him.
 
 ## License
 
-This work is dual licensed under the MIT and Apache-2.0 licenses.
-see [LICENSE-MIT](./LICENSE-MIT) and [LICENSE-APACHE](./LICENSE-APACHE) for details.
+This work is dual licensed under the ISC license. See [LICENSE](./LICENSE).
```
Comment (erickt) [inline README.md]: This would no longer be dual-licensed.

**Review by erickt — COMMENTED** [review]
Unfortunately I’m not a lawyer so I can’t give you guidance on this.

**Pull request merged**